index.html

```diff
@@ -71,14 +72,18 @@ <h1>Kanji 漢字</h1>
     </main>
     <main class="page page--create hidden" data-page="create">
         <section class="create-page__content">
-        <h3 class="create-page__h3">Add question:</h3>
-        <input class="create-input" id="input-question" type="textarea">
-        <h3 class="profile-page__h3">Add answer:</h3>
-        <input class="create-input" id="input-answer" type="text">
-        <h3 class="profile-page__h3">Add tags:</h3>
-        <input class="create-input" id="input-tags" type="text">
-        <button class="submit" type="submit">Submit</button>
-        </section>
+            <form data-js="form">
+                <fieldset class="fieldset">
+                    <label for="add-question">Add Kanji/word:</label>
+                <textarea name="question" class="create-input" id="inputQuestion" placeholder="Add Kanji/word" rows="3" cols="20" required></textarea> 
+                <label  for="add-question">Add translation/readings:</label>
+        <textarea name="answer" class="create-input" id="input-answer" type="text" placeholder="Add translation/readings" rows="3" cols="20" required></textarea> 
+        <label for="add-tags">Add tags:</label>
+        <textarea name ="tags" class="create-input" id="input-tags" type="text" placeholder="Add tags, for example level-1" rows="2" cols="20"></textarea> 
+        <button class="submit" type="submit">Create card</button>
+    </fieldset>
+    </form>
```

Comment:
Something went wrong with the formatting I think. 🤔 I'm not sure if you really need the `fieldset` element, but that is up to you. I would remove the `fieldset` element and apply the stylings to the `form` element.